### PR TITLE
Host API docs on the polymer site

### PR DESCRIPTION
**Pull request by garlicnation**
API docs available at localhost:3000/1.0/api

### CODE REVIEW

File: 1.0/api/bower.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "polymer-site",
+  "version": "0.0.0",
+  "homepage": "https://github.com/Polymer/docs",
+  "authors": [
+    "Eric Bidelman <ebidel@gmail.com>"
+  ],
+  "description": "Polymer website",
+  "keywords": [
```
Comment (ebidel) [inline 1.0/api/bower.json]: for the new site, let's remember to remove this and use the main site's bower.json

File: 1.0/api/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<!--
+Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
```
Comment (ebidel) [inline 1.0/api/index.html]: 2015

File: 1.0/api/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
```
Comment (ebidel) [inline 1.0/api/index.html]: This header looks old. It's missing the .txt extensions

File: 1.0/api/index.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
+-->
+<html>
+<head>
+
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <script src="components/webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (ebidel) [inline 1.0/api/index.html]: -lite.min.js

**Comment by ebidel** [comment]
Can you also add this to our crazy bower update script:
https://github.com/Polymer/docs/blob/master/scripts/update_bower.sh

**Comment by ebidel** [comment]
ping

**Comment by ebidel** [comment]
LGTM

**Comment by arthurevans** [comment]
Hey @garlicnation I'm trying this now, and I'm only seeing method/property names, with no docs showing up. Am I doing something wrong here, or did something break?

**Comment by surajwebworld** [comment]
I want swipe navigation drawer (finger swipe like goole play store)  can u any 1 help me plz i request you. Please hrlp me

**Pull request merged**